pallets/gear/src/manager/journal.rs

```diff
@@ -140,10 +143,22 @@ where
                 // dequeued. The other case is async init.
                 wake_waiting_init_msgs(program_id);
 
-                self.clean_reservation_tasks(program_id);
+                // If we run into `InitFailure` after real execution (not
+                // prepare or precharge) processor methods, then we are
```

Comment:
```suggestion
                // prepare or precharge processor methods), then we are
```

Reply:
Will be applied next pr just to avoid waiting one more ci